### PR TITLE
feat(authService): add async status check

### PR DESCRIPTION
**Pull request by MaximBalaganskiy**
Add async status check and fix the existing function to properly handle exceptions

### CODE REVIEW

**Comment by MaximBalaganskiy** [comment]
Fixes #400

**Comment by doktordirk** [comment]
Can you add a test and documentation? Mostly copy and paste i guess

**Comment by MaximBalaganskiy** [comment]
Done

**Comment by doktordirk** [comment]
tx, will give it a quick test hopefully soon

File: test/authService.spec.js [hunk 1]
```diff
@@ -569,6 +569,113 @@ describe('AuthService', () => {
...
+        authService.isAuthenticatedAsync()
+          .then(x => {
+            expect(x).toBe(false);
+            done();
+          });
+      });
+    });
+
+    describe('even with autoUpdateToken=true', () => {
+      it('should return boolean false', () => {
```
Comment (doktordirk) [inline test/authService.spec.js]: done missing

Comment (MaximBalaganskiy) [reply test/authService.spec.js]: fixed... interesting that the test wasn't failing

**Comment by RWOverdijk** [comment]
@doktordirk What are your findings?

**Comment by RWOverdijk** [comment]
This is old.. Probably also my fault... Closing.

**Pull request closed without merging**